manifests/install.pp

```diff
@@ -0,0 +1,9 @@
+# @api private
+class htcondor::install {
+  assert_private()
+
+  package { 'condor':
+    ensure  => 'installed',
+    require => Yumrepo['condor'],
```

Comment:
rm

Reply:
removed since we are setting order in other way now.